src/main/java/Event.java

```diff
@@ -0,0 +1,14 @@
+public class Event extends Task {
+    private final String date;
+    
+    public Event(String description, String date) {
+        super(description);
+        this.date = date;
+    }
+    
+    @Override
+    public String toString() {
+        return "[E][" + this.getStatusIcon() + "] " + this.description
+            + "(at: " + this.date + ")";
```

Comment:
instead of doing everything in a single line, perhaps consider doing it 1 by 1, then return the final string. (Avoid complicated expressions)